libbeat/common/transport/tlscommon/tls_test.go

```diff
@@ -165,6 +166,29 @@ func TestApplyWithConfig(t *testing.T) {
 	assert.Len(t, cfg.CurvePreferences, 1)
 }
 
+func TestServerConfigDefaults(t *testing.T) {
+	var c ServerConfig
+	config := common.MustNewConfigFrom([]byte(``))
+	err := config.Unpack(&c)
+	require.NoError(t, err)
+	tmp, err := LoadTLSServerConfig(&c)
+	require.NoError(t, err)
+
+	cfg := tmp.BuildModuleConfig("")
+
+	assert.NotNil(t, cfg)
+	// values not set by default
+	assert.Len(t, cfg.Certificates, 0)
+	assert.Nil(t, cfg.ClientCAs)
+	assert.Len(t, cfg.CipherSuites, 0)
+	assert.Len(t, cfg.CurvePreferences, 0)
+	// values set by default
+	assert.Equal(t, false, cfg.InsecureSkipVerify)
+	assert.Equal(t, int(tls.VersionTLS10), int(cfg.MinVersion))
```

Comment:
The _minVersion_ in 6.8 for TLS is 1.0, just [recently](https://github.com/elastic/beats/pull/10191) we have changed it for 1.1

Reply:
@graphaelli and I discussed if we should change that, but that would appear to be a breaking change.

@andrewkroh WDYT?